templates/instructor_published_policy.html

```diff
@@ -16,8 +16,18 @@
 
         <div class="col-sm-9">
 
+            <div class="row">
+                <div class="col-sm-9">
+                    <h1>Academic Integrity Policy</h1>
+                </div>
 
-            <h1>Academic Integrity Policy</h1>
+                <div class="col-sm-3" style="margin-top: 2em">
+                    <a class="btn btn-default pull-right" href="{% url 'edit_published_policy' publishedPolicy.pk %}" role="button">
+                    <i class="fa fa-envelope-o"></i>
```

Comment:
When I compare the edit button on the Canvas syllabus page to this one, I noticed the Canvas button uses the pencil icon. You might consider using `fa-pencil-square-o` icon instead of `fa-envelope-o`.